ERROR    root:test_httpRun.py:185 assert 400 == 200
 +  where 400 = <Response [400]>.status_code
INFO     root:logDecoratorl.py:39 
>>===============================正在执行第11条用例======================================>>
测试标题: 竞拍报名不填地址是否失败
请求方式: post
请求头:   {'accesstoken': 'c23f18277682d8f961c7404ea6fe5a77', 'timespan': '1660813936', 'TOKEN': '86AF2DC047BB1C1853DA37D4486B24DDB061DB08A2044CBA1B5F520F38081D109E5C3181AA98B76FCED7E4BE3C91AF7E1E7E2FAB668575B92FE968A4EA9F3227A6496BEF012D4236CE199BCF45E32D98'}
请求路径: http://110.80.137.11:8051/auction/SignUp
传参方式： params
请求内容: {"auctionid": "540389829988220559", "mainNetName": "\u4f4e\u50b2", "receiveaddr": "", "receivemark": ""}
依赖测试用例: 暂无依赖用例数据
接口响应内容: {"type":"https://tools.ietf.org/html/rfc7231#section-6.5.1","title":"One or more validation errors occurred.","status":400,"traceId":"00-a06477ec8578b85241e7d31fa001c409-7b20d5ad3281aa3d-00","errors":{"receiveaddr":["The receiveaddr field is required."]}}
接口响应时长: 0.08206 s
Http状态码: 400
断言方式：相等
期望结果：False
实际结果：{'type': 'https://tools.ietf.org/html/rfc7231#section-6.5.1', 'title': 'One or more validation errors occurred.', 'status': 400, 'traceId': '00-a06477ec8578b85241e7d31fa001c409-7b20d5ad3281aa3d-00', 'errors': {'receiveaddr': ['The receiveaddr field is required.']}}
>>==============================第11条用例执行完毕=======================================>>